packages/@lwc/wire-service/package.json

```diff
@@ -12,48 +12,41 @@
         "url": "https://github.com/salesforce/lwc/issues"
     },
     "license": "MIT",
-    "main": "dist/wire-service.cjs.js",
-    "module": "dist/wire-service.js",
-    "typings": "types/index.d.ts",
-    "scripts": {
-        "build": "rollup --config ./scripts/rollup/rollup.config.js",
-        "dev": "rollup  --config scripts/rollup/rollup.config.js --watch --no-watch.clearScreen"
+    "publishConfig": {
+        "access": "public"
     },
+    "main": "./dist/index.cjs.js",
+    "module": "./dist/index.js",
+    "types": "./dist/index.d.ts",
     "files": [
-        "dist/",
-        "types/"
+        "./dist"
     ],
+    "scripts": {
+        "build": "rollup --config ../../../scripts/rollup/rollup.config.js",
+        "dev": "rollup  --config ../../../scripts/rollup/rollup.config.js --watch --no-watch.clearScreen"
+    },
+    "nx": {
+        "targets": {
+            "build": {
+                "outputs": [
+                    "./dist"
+                ]
+            }
+        }
+    },
     "devDependencies": {
         "@lwc/engine-core": "2.43.0",
         "@lwc/shared": "2.43.0"
     },
     "lwc": {
         "modules": [
-            {
-                "name": "wire-service",
```

Comment:
Technically I'm breaking `import 'wire-service'`, but we shouldn't support this anyway because of dependency confusion. (Luckily we are not vulnerable because we [squatted the package](https://www.npmjs.com/package/wire-service).)

Reply:
Also I doubt anybody is using this.